content/induction/sprints/1/day-plan/index.md

```diff
@@ -2,14 +2,32 @@
 title = 'day-plan'
 layout = 'day-plan'
 emoji= '📝'
-menu_level = ['module']
+menu_level = ['sprint']
 weight = 3
 backlog= 'Module-Induction'
 backlog_filter= 'Week 1'
 [[blocks]]
+name="energiser"
+[[blocks]]
 name="Goals and expectations on the Software Development Course"
 src="https://cyf-pd.netlify.app/blocks/goals-and-expectations-on-the-software-development-course/readme/"
+time="80"
 [[blocks]]
 name="Google Sheet for pairing"
 src="https://cyf-pd.netlify.app/blocks/set-up-a-google-sheet-to-organise-the-pairs-for-the-next-module/readme/"
+time="15"
+[[blocks]]
+name="learning-as-community"
+src="module"
+[[blocks]]
+name="lunch"
+[[blocks]]
+name="Git workshop"
+src="https://github.com/CodeYourFuture/CYF-Workshops/blob/rewrite/git-day-1/git-day-1/README.md"
+time="120"
```

Comment:
I can't update the time for this block "Git workshop" - not sure what I'm doing wrong here. 🤔